include/dao/dao.js

```diff
@@ -710,6 +761,24 @@ module.exports = function DAOModule(pb) {
     };
 
     /**
+     * Gets all collection names
+     * @returns {DAO}
+     */
+    DAO.prototype.getAllCollections = function(cb) {
+        pb.dbm.getDb(this.dbName, function(err, db) {
+            if (util.isError(err)) {
+                return cb(err);
+            }
+            db.collections(function(err, items) {
+                if (util.isError(err)) {
+                    return cb(err)
+                }
+                cb(err, items);
+            });
+        });
+    };
```

Comment:
This function should be allowed to pass an options argument that contains a where clause to pass to the call to ```db.collections```.  This will allow ```DAO.entityExists``` to leverage the function